rir/src/ir/Compiler.cpp

```diff
@@ -1084,10 +1087,13 @@ SEXP findClosure(SEXP sym, SEXP rho) {
 
 // try to look up a closure
 bool compileWithGuess(Context& ctx, SEXP ast, SEXP fun, SEXP args_) {
+    if (!ctx.env())
+        return false;
+
     RList args(args_);
     CodeStream& cs = ctx.cs();
 
-    SEXP cls = findClosure(fun, R_GlobalEnv);  // or current env of the closure?
```

Comment:
Thanks, my bad... Those are the comments that nobody ever deals with after merging